dask_jobqueue/core.py

```diff
@@ -210,12 +214,21 @@ def job_file(self):
                 f.write(self.job_script())
             yield fn
 
+    def submit_job(self, script_filename):
+        """ Sumbits job and handles lsf exception """
+        if self.scheduler_name == 'lsf':
+            piped_cmd = [self.submit_command + '<\"' + script_filename + '\"']
+            self.shell = True
+            return self._call(piped_cmd)
+        else:
+            return self._call(shlex.split(self.submit_command) + [script_filename])
```

Comment:
Would it be possible to keep the core solution simple and instead put the LSF-specific implementation on the `LSFCluster` class?

Reply:
Ah I see what @lesteve was suggesting